products/xcp-ng.md

```diff
@@ -0,0 +1,63 @@
+---
+title: XCP-ng
+category: os
+iconSlug: xcp-ng
+permalink: /xcp-ng
+alternate_urls:
+-   /xcpng
+versionCommand: software-version
+releasePolicyLink: https://xcp-ng.org/docs/releases.html#all-releases
```

Comment:
Should be just https://xcp-ng.org/docs/releases.html, the explanations are above the list of releases.

Reply:
Changed.